docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,104 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+a number of {feature-vars} and a {depvar}, then making further predictions based 
+on the described relationship.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. In this example, the rent is the 
```

Comment:
It seems to me that you can omit everything between "rent in a city." and "Our imaginary dataset...".